src/renderer/core/api/api-service-mock.ts

```diff
@@ -3,12 +3,12 @@ import { ApiService } from './api-interface';
 import { serverTestData } from '../../model/mock/testData';
 import { of } from 'rxjs';
 
-const postConsent = (consent: ConsentAction) =>
+const postConsent = () =>
```

Comment:
Removing the function parameter without removing in the corresponding function results to errors when running `yarn tsc`. Same thing for `postMissingBrand`.

Reply:
I would suggest making them optional since we are not passing anything to the mocks when we call these functions.
Yeah the reflex to remove them was a quick-draw.

Reply:
As the parameters are required in the production code, it's more stable to keep them as is and silence the linter for test related code like this mock.